cds/webapi/C#/BlazorCDS/BlazorCDS/wwwroot/appsettings.json

```diff
@@ -0,0 +1,14 @@
+{
+  "AzureAd": {
+    "Authority": "https://login.microsoftonline.com/organizations",
+    "ClientId": "52470569-ea16-4732-aacf-8bb93ee709b9",
+    "ValidateAuthority": true
+  },
+  "CDSWebAPI": {
+    "Version": "v9.1",
+    "TimeoutSeconds": 30
+  },
+  "GDSWebAPI": {
+    "ResourceUrl": "https://globaldisco.crm.dynamics.com"
```

Comment:
`https://globaldisco.crm.dynamics.com` is a constant so it seems a bit odd to include it as a configuration.
The code in Program.cs could be simplified if it it didn't need to get this value from configuration.

Reply:
removed it from appSettings.json